### ISSUE TITLE
"Edit Page"-button in kotlin multiplatform docs is broken

### ISSUE THREAD

**Issue by jQrgen** [InsertKoinIO/koin#1502]
The "Edit Page"-button links here: https://github.com/InsertKoinIO/website/edit/master/docs/quickstart/kmm.md

Which is a 404

**Comment by arnaudgiuliani** [comment]
yep. Also all documentation is available here: https://github.com/InsertKoinIO/koin/tree/main/docs
